webext/raptor-chrome/runner.js

```diff
@@ -53,7 +53,7 @@ function getTestSettings() {
         }
         console.log("using page timeout (ms): " + pageTimeout);
 
-        if (testType == 'tp7') {
+        if (testType == 'pageload') {
           getFCP = settings['measure']['fcp'];
           if (settings['measure']['hero'].length !== 0) {
             getHero = true;
```

Comment:
Inside the web extension content it still needs to know exactly what we're measuring, not really sure how it's possible to make that more generic - when you need to provide the details of the hero element tag etc.

If you add more hero elements to the INI it will work fine, but yeah if you add other things then we need something on the webext side to recognize what that is and how to measure it... open to suggestions how to make that more generic.